server/routes/api.js

```diff
@@ -36,6 +36,15 @@ router.route('/posts/user')
       });
   });
 
+router.route('/posts/nearby')
```

Comment:
We should discuss all the endpoints, this doesnt seem to adhere to RESTful routing.